src/lib/containers/table/StandaloneQueryWrapper.tsx

```diff
@@ -136,52 +136,46 @@ const StandaloneQueryWrapper: React.FunctionComponent<StandaloneQueryWrapperProp
 
                     return (
                       <>
-                        {title ? (
```

Comment:
Removed logic that when a title was not supplied, then nothing is shown.  Verified in portals project that all configurations of StandaloneQueryWrapper give a title (so this should have no impact on portals).